source/server/server.cc

```diff
@@ -15,6 +15,7 @@
 #include "envoy/upstream/cluster_manager.h"
 
 #include "common/api/api_impl.h"
+#include "common/common/backward.h"
```

Comment:
This was a relic of previous testing.  I was going to do the signal handler in a separate PR, I had a few questions about how to get the conditional option through the build system.